src/MensaWidget/MWDataGettingWrapper.class.st

```diff
@@ -53,19 +53,23 @@ MWDataGettingWrapper class >> canteenDict: anObject [
 
 {
 	#category : #dataPulling,
-	#'squeak_changestamp' : 'Max 7/5/2024 09:37'
+	#'squeak_changestamp' : 'Oscar 7/11/2024 14:41'
 }
 MWDataGettingWrapper class >> getMealsOnDate: aDate [
 
 	| canteens canteenDict json |
 	
 	canteens := OrderedCollection new.
-	self canteenDict isEmpty ifTrue: [^ canteens add: MWCanteen emptyCanteen;
-		yourself].
+	self canteenDict isEmpty 
+		ifTrue: [^ canteens add: MWCanteen emptyCanteen; yourself].
 	canteenDict := self canteenDict.
-	canteenDict keys do: [:id |
+	canteens := canteenDict keys collect: [:id |
 		json := MWApiWrapper getMealsFrom: id on: aDate using: WebClient.
-		canteens add: (MWCanteen newWithId: id  Name: (canteenDict at: id) Date: aDate Meals: (MWMealCleaner clean: (MWParser put: json in: MWMeal)))].
+		canteens add: (MWCanteen 
+							newWithId: id  
+							Name: (canteenDict at: id) 
+							Date: aDate 
+							Meals: (MWMealCleaner clean: (MWParser put: json in: MWMeal)))].
```

Comment:
too many identations?

Reply:
It's a trade of. We can make it a wide and short method or a longer one that is easier to read. 

But if you don't like it I can change it back.

Reply:
No I like it that way I mean it looks like too many tabs

Reply:
ahh I see...